keyboards/annepro2/c15/rules.mk

```diff
@@ -0,0 +1,41 @@
+# Anne Pro 2
+SRC = \
+	matrix.c \
+	hardfault_handler.c \
+	annepro2_ble.c \
+	qmk_ap2_led.c \
+	protocol.c
+
+LAYOUTS +=
+
+# MCU
+MCU = cortex-m0plus
+ARMV = 6
+USE_FPU = no
+MCU_FAMILY = HT32
+MCU_SERIES = HT32F523xx
+MCU_LDSCRIPT = HT32F52342_ANNEPRO2
+MCU_STARTUP = ht32f523xx
+
+BOARD = ANNEPRO2
+
+OPT_DEFS = -Wno-unused-function -fdump-rtl-dfinish -fstack-usage
+#EXTRALDFLAGS = -Wl,--print-memory-usage
+
+# Options
+
+# Keys
+CUSTOM_MATRIX = yes
```

Comment:
This should be converted to matrix lite, if/when possible. 

https://docs.qmk.fm/#/custom_matrix?id=lite